### ISSUE TITLE
bug: remix │ |- TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".css" node_modules/katex/dist/katex.min.css

### ISSUE THREAD

**Issue by bakikucukcakiroglu** [steven-tey/novel#432]
### Provide environment information

  System:
    OS: macOS 14.5
    CPU: (11) arm64 Apple M3 Pro
    Memory: 172.17 MB / 18.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.15.1 - ~/.nvm/versions/node/v20.15.1/bin/node
    Yarn: 1.22.22 - /opt/homebrew/bin/yarn
    npm: 10.7.0 - ~/.nvm/versions/node/v20.15.1/bin/npm
    pnpm: 9.6.0 - ~/.nvm/versions/node/v20.15.1/bin/pnpm

### Describe the bug

I got this error when I try to use novel in my remix project. I used the same code in my next project and everything was perfect. 

```
13:12:39 │            remix │ 1:12:39 PM [vite] Error when evaluating SSR module
/app/routes/app.products.$productId.editor/route.tsx: failed to import "novel/extensions"
13:12:39 │            remix │ |- TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".css" for
```

### Link to reproduction

-

### To reproduce

-

**Comment by elianiva** [comment]
i think this is caused be the css import specified here:
https://github.com/steven-tey/novel/blob/ecdb5d2ff5d3e6ab5f5af95b5a81de3ab4e8a0e9/packages/headless/src/extensions/mathematics.ts#L1
which gets bundled when it's built, and node tries to import a `.css` file when doing ssr

**Comment by someguy9** [comment]
Same issue here, not sure how to resolve

**Comment by bakikucukcakiroglu** [comment]
It looks like the library is dead. I wouldn't recommend using it for a production level work.

**Comment by phyohtetarkar** [comment]
Yes this library isn't active. Here I re-wrote the [block-editor](https://github.com/phyohtetarkar/hope-elearning-web/tree/main/packages/block-editor) using Tiptap + Tailwind + Shadcn as a separate package for my project. I think it will be useful for you.

**Comment by jt6677** [comment]
I first started using novel in August 2023. I wish I did not waste so much time on this repo. It started out as a ui component you can easily modify. They tried so hard to force it into a stand alone package.  All the complexity added on top of all the nextjs stuff.

**Comment by yll0rd** [comment]
@bakikucukcakiroglu 

This is what worked for me: 
```
import dynamic from "next/dynamic";
export const Editor = dynamic(() => import("@/components/novel-editor/advanced-editor"), { ssr: false }); 
```
